portfolio/src/main/java/com/google/sps/servlets/RestaurantDataServlet.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.data.Restaurant;
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns restaurant data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
+@WebServlet("/restaurant-data")
+public class RestaurantDataServlet extends HttpServlet {
+
+  private Collection<Restaurant> restaurants;
+  private String my_region = "California";
+  private static final Map<Integer, String> NUMBERS = ImmutableMap.of(
+    1, "one", 2, "two", 3, "three");
```

Comment:
nit: generally the Java style guide recommends placing static fields above non-static fields. 

Copy-pasta from the style guide:

Recommended general approach
There are no perfect, deterministic solutions here; you will be using a great deal of personal judgment.

First, think only about the API (public and protected members) and the state (ordinary instance fields) of your type. Work out how to arrange those without worrying about any other members just yet.

We recommend a high-level ordering like this (notice that visible static fields and static methods are grouped together before instance fields):
Visible static fields (usually final)
Visible static methods
"Regular" instance fields (comprising object state, not derived)
Constructors (whether visible or not)
Visible instance methods